README.md

```diff
@@ -173,6 +187,98 @@ Kind: *Trusted Indication*.
 
 Errors:
 
+### `HrmpNewChannelOpenRequest`
+
+A message to notify about a new incoming HRMP channel. This message is meant to be sent by the
+relay-chain to a para.
+
+- `sender: u32 (Compact)`: The sender in the to-be opened channel. Also, the initiator of the channel opening.
+- `max_message_size: u32 (Compact)`: The maximum size of a message proposed by the sender.
+- `max_capacity: u32 (Compact)`: The maximum number of messages that can be queued in the channel.
+
+Safety: The message should originate directly from the relay-chain.
+
+Kind: *System Notification*
+
+### `HrmpChannelAccepted`
+
+A message to notify about that a previously sent open channel request has been accepted by
+the recipient. That means that the channel will be opened during the next relay-chain session
+change. This message is meant to be sent by the relay-chain to a para.
+
+- `recipient: u32 (Compact)`: The recipient in the to-be opened channel.
+
+Safety: The message should originate directly from the relay-chain.
+
+Kind: *System Notification*
+
+Errors:
+
+### `HrmpChannelClosing`
+
+A message to notify that the other party in an open channel decided to close it. In particular,
+`inititator` is going to close the channel opened from `sender` to the `recipient`. The close
+will be enacted at the next relay-chain session change. This message is meant to be sent by
+the relay-chain to a para.
+
+- `initiator: u32 (Compact)`: The initiator of the channel closing.
+- `sender: u32 (Compact)`: The sender in the to-be closed channel.
+- `recipient: u32 (Compact)`: The recipient in the to-be closed channel.
+
+Safety: The message should originate directly from the relay-chain.
+
+Kind: *System Notification*
+
+Errors:
+
+### `HrmpInitOpenChannel`
+
+Initiate opening a channel from a parachain to a given recipient with given channel
+parameters.
+
+- `recipient: u32 (Compact)`, - The recipient in the to-be opened channel.
+- `proposed_max_capacity: u32 (Compact)` - specifies how many messages can be in the channel at once.
+- `proposed_max_message_size: u32 (Compact)` - specifies the maximum size of any of the messages.
+
+These numbers are a subject to the relay-chain configuration limits.
+
+The channel can be opened only after the recipient confirms it and only on a session
+change.
+
+Safety: The message should originate directly from the para-chain.
+
+Kind: *Instruction*.
+
+### `HrmpAcceptOpenChannel`
+
+Accept a pending open channel request from the given sender.
+
+The channel will be opened only on the next session boundary.
+
+- `sender: u32 (Compact)`: The sender in the to-be opened channel. Also, the initiator of the channel opening.
+
+Safety: The message should originate directly from the para-chain.
```

Comment:
```suggestion
Safety: The message should originate directly from the parachain.
```

I think we spell it as parachain rather than para-chain. (There are other places as well)